### PR TITLE
More content on 3d models and audio

### PR DESCRIPTION
**Pull request by nearnshaw**

### CODE REVIEW

**Comment by nearnshaw** [comment]
@menduz , please review and merge

File: _posts/documentation/building-scenes/2018-01-03-create-scene.md [hunk 1]
```diff
@@ -13,7 +13,13 @@ tag: introduction
...
 ## Before you begin
 
-Please make sure you first install the CLI tools. See the [Installation Guide]({{ site.baseurl }}{% post_url /building-scenes/2018-01-01-installation-guide %}) for details on how to do this.
+Please make sure you first install the CLI tools. In Mac OS, you do this by running the following command:
+
+```bash
+npm install -g decentraland
+```
+
+See the [Installation Guide]({{ site.baseurl }}{% post_url /building-scenes/2018-01-01-installation-guide %}) for more details and specific instructions for Windows and Linux systems.
```
Comment (eordano) [inline _posts/documentation/building-scenes/2018-01-03-create-scene.md]: correct url is `/documentation/building-scenes/2018-01-01-installation-guide`

File: _posts/sdk-reference/2018-01-21-scene-content-guide.md [hunk 2]
```diff
@@ -38,12 +38,143 @@ See [Entity interfaces]({{ site.baseurl }}{% post_url /sdk-reference/2018-06-21-
 
 > Tip: When editing the code via a IDE (like Visual Studio Code), you can see the list of components supported by a type of entity. Typically, this is done by placing the cursor in the entity and pressing *Ctrl + Space bar*.
 
+## Entity ppositioning
```
Comment (eordano) [inline _posts/sdk-reference/2018-01-21-scene-content-guide.md]: Typo

**Review by eordano — APPROVED** [review]
Needs some more changes but it's good to go

**Pull request merged**